examples/demo13.py

```diff
@@ -93,11 +93,11 @@ def main(argv):
     logger.debug('Read in WFIRST imaging filters.')
 
     logger.info('Reading from a parametric COSMOS catalog.')
-    # Read in a galaxy catalog - just a random subsample of 100 galaxies for F814W<23.5 from COSMOS.
+    # Read in a galaxy catalog - just a random subsample of 100 galaxies for F814W<25.2 from COSMOS.
```

Comment:
Why is this changed?  Did you locally change the contents of ```real_galaxy_catalog_example_fits.fits``` to correspond to <25.2?  I don't see the change here.  If you didn't change the file, then all changes to the comments here are incorrect.